splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -2316,6 +2322,64 @@
       },
       "required": ["title", "services"],
       "additionalProperties": false
+    },
+    "IntervalEntity": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "const": "interval"
+        },
+        "field": {
+          "$ref": "#/definitions/Field"
+        },
+        "label": {
+          "type": "string"
+        },
+        "defaultValue": {
+          "$ref": "#/definitions/IntervalValue"
+        },
+        "options": {
+          "anyOf": [
```

Comment:
that anyOf seems unnecessary , as you passed only one possibility

Reply:
You are right. I forgot to remove it.